kornia/filters/filter.py

```diff
@@ -15,7 +16,8 @@ def compute_padding(kernel_size: Tuple[int, int]) -> List[int]:
 
 
 def filter2D(input: torch.Tensor, kernel: torch.Tensor,
-             border_type: str = 'reflect') -> torch.Tensor:
+             border_type: str = 'reflect',
+             normalized: bool = False) -> torch.Tensor:
```

Comment:
In boxblur is true by default. Why false here?

Reply:
Because everywhere else, is it us who set defaults. 
For filter2D, I believe, user would provide the exactly filter she wants to convolve with. I think, don`t change the filter be default is good thing